webapp/src/app/course-welcome/course-welcome.component.ts

```
@@ -165,7 +431,7 @@
           CourseAction.setCurrentCourse({ selcetedCourse: course })
         );
         this.store.dispatch(CourseAction.setCourseId({ courseId: course._id }));
-        this.socket.emit("leave", "course:"+course._id);
+        this.socket.emit('leave', 'course:' + course._id);
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)